person.rb

```diff
@@ -10,12 +10,17 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
+
```

Comment:
- [ ] As suggested in the `add_rental` method of the `Book` class, it's not a guaranteed way to accept a `rental` and add it to the rental list, we need to separately receive the `(book, date)` parameters and create a new rental inside the `add_rental` method by adding the person as `self`. Please refer to the previous example I left in the `Book` class.